lib/v2/aiea/index.js

```diff
@@ -0,0 +1,39 @@
+const buildData = require('@/utils/common-config');
+
+module.exports = async (ctx) => {
+    const link = 'http://www.aiea.org/0504';
```

Comment:
Please use HTTPS

Reply:
The website does not use tls, should still change to https?